config/config.go

```diff
@@ -560,6 +560,11 @@ func FindConfigFilesInPath(rootPath string, terragruntOptions *options.Terragrun
 		}
 
 		if isTerragruntModule {
+			// Skip Module if search file is not found in layer
+			if terragruntOptions.LayerHasFile != "" && !util.FileExists(path+terragruntOptions.LayerHasFile) {
```

Comment:
**suggestion (non-blocking):** add a log in debug 🤷 

```golang
logger.Debugf("Skipping module %s as file %s does not exist", file.Name(), path+terragruntOptions.LayerHasFile)
```

Reply:
Agreed

Reply:
logs sample:

![Capture d’écran 2022-12-01 à 16 39 42](https://user-images.githubusercontent.com/3822295/205095362-ec12ee7c-c0b4-4b4b-9159-96c81954fd06.png)